README.md

```diff
@@ -1,3 +1,517 @@
+
+      target 'MyApp' do
+        # As we use Swift, ensure that `use_frameworks` is enabled.
+        use_frameworks!
+
+        # Specific iOS platform we are targetting
+        platform :ios, '8.0'
+
+        # Point to the installed version
+        pod 'RNDateTimePicker', :path => '../node_modules/react-native-datetimepicker/RNDateTimePicker.podspec'
+
+        # React/React-Native specific pods
+        pod 'React', :path => '../node_modules/react-native', :subspecs => [
+          'Core',
+          'CxxBridge',      # Include this for RN >= 0.47
+          'DevSupport',     # Include this to enable In-App Devmenu if RN >= 0.43
+          'RCTText',
+          'RCTNetwork',
+          'RCTWebSocket',   # Needed for debugging
+        ]
+
+        # Explicitly include Yoga if you are using RN >= 0.42.0
+        pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+
+        # Third party deps podspec link
+        pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+        pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+        pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+      end
+      ```
+  4. Run `pod install` inside the same folder where the `pod` file was created
+  5. Open the `.xcworkspace` file that was created and run
+
+#### Android
+
+1. Add the following lines to `android/settings.gradle`:
+   
+   ```gradle
+   include ':react-native-datetimepicker'
+   project(':react-native-datetimepicker').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-datetimepicker/android')
+   ```
+
+2. Add into `android/gradle.properties`:
+   
+   ```gradle
+   android.useAndroidX=true
+   android.enableJetifier=true
+   ```
+
+3. Add the compile line to the dependencies in `android/app/build.gradle`:
+
+   ```gradle
+   dependencies {
+       ...
+       implementation project(':react-native-datetimepicker')
+   }
+   ```
+
+4. Add the import and link the package in `MainApplication.java`:
+
+   ```java
+   import com.reactcommunity.rndatetimepicker.RNDateTimePickerPackage; // <-- add this import
+
+   public class MainApplication extends Application implements ReactApplication {
+       @Override
+       protected List<ReactPackage> getPackages() {
+           return Arrays.<ReactPackage>asList(
+               new MainReactPackage(),
+               new RNDateTimePickerPackage() // <-- add this line
+           );
+       }
+   }
+   ```
+
+## Tests
+
+### Jest
+
+```
+$ npm install
+$ npm run test
+```
+
+### Detox
+
+Detox is a gray box end-to-end testing and automation library for mobile apps.
+- [Dependencies required](https://github.com/wix/Detox/blob/master/docs/Introduction.GettingStarted.md#step-1-install-dependencies)
+
+For cleaning all the detox builds just run `npm run detox:clean`.
+
+#### iOS
+
+- debug:
+  ```sh
+  npm install
```

Comment:
rather than repeating install each time perhaps add a section on how to contribute to the module in general, with `clone`, `install` and `test` instructions